### DISCUSSION TITLE
gRPC FUSE vs. VirtioFS on M-based MACs

### DISCUSSION THREAD

**Discussion by tomasmarcik** [wardenenv/warden#657] [Help & Support]
Hey, I know that warden puts mutagen in place to improve file performance which is critical specially for Magento 2 development, however has anyone experimented with switching from default gRPC FUSE to VirtioFS file sharing in Docker settings? Does it realistically improve performance? How does it affect mtuagen integration?

If anyone wants to share experience from different hardware, it's welcomed but I am most curious about M1/M2 CPUs on Macs (personally using M1 Max).

I have looked around internet and there are mixed reactions to that. From the PHP world, it's more to the positive side, but I haven't found anything discussing this topic with using Warden.

Thank youf or sharing your experience!

**Comment by bap14** [comment]
I've been using virtioFS since it was available. No issues running warden with it. I've also still been letting Mutagen manage the file sync, but haven't had any challenges with it.
